charts/px-backup/values.yaml

```
@@ -0,0 +1,58 @@
+namespace: px-backup
+
+persistentStorage:
+  enabled: false
+  storageClassName:
+
+storkRequired: false
+ingressControllerSetup: false
+centralEndpoint: None
+
+oidc:
+  centralOIDC:
+    enabled: true
+    defaultUsername: admin
+    defaultPassword: admin
+    defaultEmail: admin@portworx.com
+    keyCloakBackendPassword: keycloak
+    clientId: "pxcentral"
+    clientSecret: "dummy"
+
+  externalOIDC:
+    enabled: false
+    clientID: "test"
+    clientSecret: "test"
+    endpoint: "test"
+
+pxbackup:
+  enabled: true
+  orgName: portworx
+
+images:
+  customRegistryEnabled: false
+  registry: docker.io
+  repo: portworx
+  pullSecrets: docregistry-secret
```

This needs to be an array
```suggestion
  pullSecrets:
    - docregistry-secret
```

---

Do we need to provide multiple secrets support? for the custom registry setup user will have the only secret to pulling all images from registry right?

---

Added support of multiple secrets.